code/controllers/ExtensibleSearchAPI.php

```diff
@@ -93,22 +94,43 @@ public function toggleSuggestionApproved($request) {
 	 *	@URLparameter term <{SEARCH_TERM}> string
 	 *	@return JSON
 	 */
-
 	public function getSuggestions($request) {
 
-		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') && ($suggestions = $this->service->getSuggestions($request->getVar('term'), $request->getVar('page')))) {
+		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions')) {
+			$suggestions = $this->service->getSuggestions($request->getVar('Search'), $request->getVar('page'));
```

Comment:
There were two sides to this, one being that the request for suggestions may have not been valid (either they entered a search query with less than 3 characters or a matching search page was not found). In which case, returning an empty result set doesn't really make sense. The other side was regarding security, and that a user may not have access to view the search page, in which case a 404 makes more sense.

I guess the question I'm trying to ask here is, did this need to be changed in regards to typeahead, or could it be considered a separate issue?